protocols/migration/participants.py

```diff
@@ -7,6 +7,60 @@
 from protocols.migration import BaseMigration
 
 
+class MigrationParticipants100To104SNAPSHOT(BaseMigration):
+    old_model = participant_1_0_0
+    new_model = participant_1_0_4_SNAPSHOT
+
+    def migrate_cancer_participant(self, cancer_participant):
```

Comment:
I think VersionControl migration is missing.

Reply:
Good point @priesgo I'll make sure to add an explicit test